### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bgravato**
Add known issue: only works with silverstripe/blog 1.x
Add note and installation instructions on how to workaround that issue

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,20 @@
 
 ##Installation Instructions
 
-    composer require silverstripe/wordpressimport 0.3.*
+    composer require thiagovandieten/export-wordpresstosilverstripe *
```
Comment (camfindlay) [inline README.md]: this should be the link to the official repo rather than a fork.

Comment (bgravato) [inline README.md]: Agreed, but silverstripe/wordpress doesn't show up on http://addons.silverstripe.org/add-ons
I can't find it with composer either (don't know why).
The only module I could find and install was thiagovandieten/export-wordpresstosilverstripe, so I thought that would be the "official" now...

Comment (camfindlay) [inline README.md]: True. I need to register this on packagist. I've recently moved the repo so was waiting.

Comment (camfindlay) [inline README.md]: registering this now under camfindlay/silverstripe-wordpressimport

**Pull request merged**